### PR TITLE
Make example working as instructed in ReadMe

### PR DESCRIPTION
**Pull request by davincho**
In order to run be able to try lost-pixel out locally, I had to do these changes.

### CODE REVIEW

**Comment by d-ivashchuk** [comment]
Thanks for the contribution @davincho 🤝 

Mind bumping the lost-pixel version in other examples as well? The adjacent storybook example(previous version) could also use copy-pasted commands that you have rewritten already

**Comment by davincho** [comment]
Updated all examples in https://github.com/lost-pixel/lost-pixel-examples/pull/2/commits/88f9634caa3c736e9fecfbc5a1931f12a1759149

File: example-ladle/lostpixel.config.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { CustomProjectConfig } from "lost-pixel";
 
 export const config: CustomProjectConfig = {
   ladleShots: {
-    ladleUrl: "http://172.17.0.1:61000",
+    ladleUrl: "http://127.0.0.1:61000",
```
Comment (d-ivashchuk) [inline example-ladle/lostpixel.config.ts]: this will be true for local run, but on github actions it will fail as there they have a specific IP

**Review by d-ivashchuk — CHANGES_REQUESTED** [review]
One little change for reverting the ladle URL. Other than that looks fantastic, thanks!

File: example-ladle/lostpixel.config.ts (hunk 1 above)
Comment (davincho) [reply example-ladle/lostpixel.config.ts]: Ok, but then it won't work locally.

```suggestion
    ladleUrl: "http://172.17.0.1:61000",
```

Comment (d-ivashchuk) [reply example-ladle/lostpixel.config.ts]: yeah, that could be improved somehow. Will think about how we can educate better about it

**Pull request merged**